app/controllers/AjaxController.php

```diff
@@ -19,11 +19,17 @@ public function get_data() {
 				$this->email = $_POST['email'];
 				$this->password = $_POST['password'];
 				$this->date = $_POST['date'];
+				$this->mobile = $_POST['mobile'];
 				$this->gender = $_POST['gender'];
 				$this->address = $_POST['address'];
 
 				print_r($this);
 			}
+
+			public function register() {
+				$user = new User();
+				$user->name = 
```

Comment:
This should show an error: `$user->name = ` Why does no value assigned to it?

Reply:
Because I did that much for yesterday :(